telethon/client/telegrambaseclient.py

```diff
@@ -215,6 +215,7 @@ def __init__(
             connection: 'typing.Type[Connection]' = ConnectionTcpFull,
             use_ipv6: bool = False,
             proxy: typing.Union[tuple, dict] = None,
+            local_addr=None,
```

Comment:
Could you also add this parameter to the docstring above? Borrowing what Python has to say about it should be enough (correct me if I'm wrong on the second sentence though, but it's often useful to tell the users when they would want to use something):

> Local host address used to bind the socket to locally. You only need to use this if you have multiple network cards and want to use a specific one.

Reply:
Ups, sure. I just added it.